xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd

```diff
@@ -85,169 +86,7 @@ Rail transport, Roads and Road transport
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
-	<!-- =====AdministrativeRole Identifiers=============================================================== -->
-	<xsd:simpleType name="StakeholderRoleTypeEnumeration">
```

Comment:
For reference. This went into ./netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd: